README.md

```diff
@@ -29,6 +29,9 @@ To generate API code and mocks, use `make generate`  This requires installing mo
 
 ## Running
 
+Prerequisites:
+* `virt-install`
```

Comment:
This is more of a testing dependency right?

Reply:
Yes

Reply:
OK so can we move this to the test README as its not actually a dep of the project :pray:

Reply:
I guess the README steps would require it, my bad.